src/xenia/cpu/processor.cc

```diff
@@ -900,8 +902,8 @@ void Processor::Continue() {
     assert_always("cancel stepping not done yet");
   }
   execution_state_ = ExecutionState::kRunning;
-  ResumeAllBreakpoints();
   ResumeAllThreads();
+  ResumeAllBreakpoints();
```

Comment:
this would cause some sore of breakpoint loop... resuming after resuming the thread seems to fix the issue where it freezes upon continuing from a breakpoint